addons/point_of_sale/static/src/app/store/product_configurator_popup/product_configurator_popup.js

```diff
@@ -108,18 +112,27 @@ export class ProductConfiguratorPopup extends Component {
 
     setup() {
         useSubEnv({ attribute_components: [] });
-    }
+        this.pos = usePos();
+        this.ui = useState(useService("ui"));
+        this.inputArea = useRef("input-area");
+        this.state = useState({
+            product: this.props.product,
+            payload: this.env.attribute_components,
+        });
 
-    get attributeLines() {
-        return this.props.product.attribute_line_ids.filter((attr) => attr.attribute_id);
-    }
+        onMounted(() => {
+            this.computeProductProduct();
+        });
```

Comment:
```suggestion
        this.computeProductProduct();
```
Can't you do this without the onMounted?